src/main.cpp

```diff
@@ -0,0 +1,146 @@
+#include <filesystem>
+
+#include <spdlog/spdlog.h>
+
+#include "silo/common/overloaded.h"
+#include "silo/common/panic.h"
+#include "silo/config/config_repository.h"
+#include "silo/config/preprocessing_config.h"
+#include "silo/config/runtime_config.h"
+#include "silo/database.h"
+#include "silo/preprocessing/preprocessing_exception.h"
+#include "silo/preprocessing/preprocessor.h"
+#include "silo_api/api.h"
+#include "silo_api/logging.h"
+
+namespace {
+
+/// Does not throw exceptions
+int runPreprocessor(const silo::config::PreprocessingConfig& preprocessing_config) {
+   // TODO (#656): move body of siloPreprocessing to preprocessing.{h,cpp} #656
```

Comment:
Why not simply do it now? This function is new in this PR anyway. (But no strong opinion here)

Reply:
To get PRs merged as quickly as possible.